### PR TITLE
Adds ability to retrieve a company's logo

### PR DESCRIPTION
**Pull request by rodolfobandeira**
```ruby
IEX::Resources::Logo.get('MSFT')
```

```
https://storage.googleapis.com/iex/api/logos/MSFT.png
```

### CODE REVIEW

**Comment by dblock** [comment]
Add a line to https://github.com/dblock/iex-ruby-client/blob/master/CHANGELOG.md and this is good to go! Thanks.

**Comment by rodolfobandeira** [comment]
@dblock Added. Thanks!

File: spec/iex/resources/logo_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe IEX::Resources::Logo do
+  context "retrieve company's logo", vcr: { cassette_name: 'logo/msft' } do
+    subject do
+      IEX::Resources::Logo.get('MSFT')
+    end
+
+    it 'retrieves a logo' do
+      expect(subject['url']).to eq 'https://storage.googleapis.com/iex/api/logos/MSFT.png'
```
Comment (dblock) [inline spec/iex/resources/logo_spec.rb]: I missed that it's returning a structure. This should actually return a 1st class object called `Logo` with a property called `url`. See how it's done for OHLC.

Comment (rodolfobandeira) [reply spec/iex/resources/logo_spec.rb]: Good catch! I didn't notice that. So, now it should be good:

```ruby
    9:     it 'retrieves a logo' do
   10:       require 'byebug'
   11:       byebug
=> 12:       expect(subject.url).to eq 'https://storage.googleapis.com/iex/api/logos/MSFT.png'
   13:     end
   14:   end
   15: end
(byebug) subject
#<IEX::Resources::Logo url="https://storage.googleapis.com/iex/api/logos/MSFT.png">
(byebug) subject.url
"https://storage.googleapis.com/iex/api/logos/MSFT.png"
(byebug)
```

File: README.md [hunk 2]
```diff
@@ -83,6 +83,18 @@ company.company_name # 'Microsoft Corporation'
...
+
+Fetches company logo for a symbol.
+
+```ruby
+logo = IEX::Resources::Logo.get('MSFT')
+
+logo.url # 'https://storage.googleapis.com/iex/api/logos/MSFT.png'
+```
+
+See [#logo](https://iextrading.com/developer/docs/#logo) for detailed documentation or [logo.rb](lib/iex/resources/logo.rb) for returned field.
```
Comment (dblock) [inline README.md]: "returned fields" (plural), even though single field right now probably.

File: spec/iex/resources/logo_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe IEX::Resources::Logo do
+  context "retrieve company's logo", vcr: { cassette_name: 'logo/msft' } do
```
Comment (dblock) [inline spec/iex/resources/logo_spec.rb]: minor, "retrieves company's logo", just more idiomatic

**Comment by rodolfobandeira** [comment]
@dblock Fixed typos and rebase the latest master. Thanks!

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,7 @@
 * [#5](https://github.com/dblock/iex-ruby-client/pull/5): Added `IEX::Resources::OHLC#get` - [@jromanovs](https://github.com/jromanovs). 
 * [#5](https://github.com/dblock/iex-ruby-client/pull/5): Added `IEX::Resources::OHLC#market` - [@jromanovs](https://github.com/jromanovs). 
 * [#9](https://github.com/dblock/iex-ruby-client/pull/9): Added `IEX::Errors::ClientError` - [@rodolfobandeira](https://github.com/rodolfobandeira).
+* [#10](https://github.com/dblock/iex-ruby-client/pull/10): Added `IEX::Resources::Logo#get` - [@rodolfobandeira](https://github.com/rodolfobandeira).
```
Comment (dblock) [inline CHANGELOG.md]: I missed this, this changelog entry and the one above belongs in 0.3.4 above, not in the released 0.3.3.

Comment (rodolfobandeira) [reply CHANGELOG.md]: Fixed. Please let me know if that is in the correct format. I guess you'll add the new entry `0.3.5` and edit the date when you release the `0.3.4` right?

Thanks again!

Comment (dblock) [reply CHANGELOG.md]: Right.

**Comment by dblock** [comment]
Merged, thanks!

**Pull request merged**